src/StatsdClient/StatsdConfig.cs

```diff
@@ -5,6 +5,7 @@ public class StatsdConfig
         public string StatsdServerName { get; set; }
         public int StatsdPort { get; set; }
         public int StatsdMaxUDPPacketSize { get; set; }
+        public int MaxUnixDomainSocketPacketSize {get; set;} = 2048;
```

Comment:
Could you keep consistent with the naming of the  other config options ? ie `Statsd...`